locksmith/package.json

```diff
@@ -50,6 +50,7 @@
     "apollo-link-http": "1.5.17",
     "apollo-server": "3.6.7",
     "apollo-server-express": "2.25.4",
+    "bignumber.js": "9.1.0",
```

Comment:
Can we use the Ethers version please? These are different libs and I really want to avoid using multiple libs for big numbers!